rotkehlchen/tests/api/test_balances.py

```diff
@@ -307,7 +307,12 @@ def test_query_all_balances_ignore_cache(
         msg = 'call count should increase since cache should have been ignored'
         assert all(fn.call_count == 2 for fn in function_call_counters), msg
         msg = 'etherscan call count should have doubled after forced token detection'
-        expected_count = full_query_etherscan_count * 2 - len(ethereum_accounts)
+        # NB: Subtract `get_eth2_staked_amount()` calls not done due to
+        # REQUEST_DELTA_TS constraint for cache ignoring request case.
+        eth2_calls_count = 14
```

Comment:
@LefterisJP Please don't think I'm happy about this hard coded number. I'm aware there are two addresses and I've followed the calls but I'm still unsure if there is a linear relation between number of addresses and calls (e.g. 1 address = 7 calls, 2 addresses = 14 calls, etc.).
I've tried changing the parametrized `number_of_eth_accounts` but the test fails for other reasons.
Feel free to change this statement for something more meaningful!

Reply:
no worries victor, we can figure it out later. Thanks a lot for looking at this. All is green :)